src/controllers/AdminController.php

```diff
@@ -334,12 +336,45 @@ class="fb-cta cta--yellow"
     $timer_on = $timer->getValue() === '1';
     $timer_off = $timer->getValue() === '0';
 
-    if ($start_ts->getValue() === '0') {
-      $start_ts = tr('Not started yet');
-      $end_ts = tr('Not started yet');
+    if ($start_ts->getValue() !== '0' && $start_ts->getValue() !== 'NaN') {
+      $game_start_ts = $start_ts->getValue();
```

Comment:
If XHP supports it, then the game_start variables; something like:
```php
$game_start = [
     'year' => '0',
     'month' => '0',
     'day' => '0',
     // etc
];
```

There is also `getdate()`, which should return all the data you want as an array.